docs/pages/starter-kit/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Starter Kit'
+---
+
+# Starter kit
+
+The starter kit is an example of an application running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
+
+This application is fork-able on this repository: https://github.com/liteflow-labs/marketplace-template.
+
+## Features included
+
+The Marketplace template includes the following features:
+
+- Open collections (User-generated content)
+  - Mint ERC-721
+  - Mint ERC-1155
+  - Creator's royalties support
+- Featured NFTs on the home page
+- Search system
+- Explore NFTs with filtering and sorting systems
+- Native currency support
+- List NFTs on sale
+  - Fixed price listing
+  - Timed auction listing
+- Make offers on NFTs
+  - Open offer
+  - Timed auction offer
+- Purchase NFTs
+- User profile
+  - Account
+    - Profile edition
+    - Wallet page
+  - NFTs "on sale"
+  - NFTs "owned"
+  - NFTs "created"
+  - Offers ("bids") management
+  - Trades history
+  - Listings ("offers") management
+- User verification system (verified status)
+- Report system
+  - Report a user
+  - Report an NFT
+- Notifications
+  - In-app notifications
+  - Email notifications
```

Comment:
This is done by the infra
```suggestion
```

Reply:
This is from the readme of the template https://github.com/liteflow-labs/marketplace-template/blob/main/README.md
Also I would not be against putting this here and push to handle email notifications at the application level ;)